Button/Button.jsx

```diff
@@ -15,15 +16,22 @@ class Button extends MaterialComponent {
     super();
     this.componentName = "button";
     this._mdcProps = ["dense", "raised", "compact", "unelevated", "stroked"];
+    this.themeProps = ["primary", "secondary"];
   }
   componentDidMount() {
     super.attachRipple();
   }
   materialDom(props) {
     const ButtonElement = props.href ? "a" : "button";
+    let className = "";
+    this.themeProps.forEach(themeProp => {
+      if (themeProp in props && themeProp !== false)
```

Comment:
You forgot that `themeProp` is a possible key of a prop in `props`. It's also a string and it wouldn't make sense to compare a string to a boolean. Change `themeProp !== false` to `props[themeProp] !== false`.

Reply:
Whoops!